### PR TITLE
Added TLS cluster how to

### PR DESCRIPTION
**Pull request by yuce**
## Overview

This PR adds TLS cluster how to.
Currently requires, so blocked by: https://github.com/pilosa/pilosa/pull/889 and possibly https://github.com/pilosa/pilosa/pull/892

The required binary may be compiled from source after merging those branches or downloading one of:
Linux amd64: https://storage.googleapis.com/pilosa/pilosa-tls-linux
MacOS: https://storage.googleapis.com/pilosa/pilosa-tls-macos

## Pull request checklist

- [x] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [x] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [x] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

**Comment by yuce** [comment]
Followed the tutorial on Linux, WSL and MacOS; it works fine.

File: docs/howtos.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+[gossip]
+seed = "01.pilosa.local:15000"
+port = 17000
+key = "pilosa.local.gossip32"
+```
+
+Here is some explanation of the configuration items:
+* `data-dir` points to the directory where the Pilosa server writes its data. If it doesn't exist, the server will created it.
+* `bind` is the address which the server listenes to for incoming requests. The address is composed of three parts, the scheme, host and port. The default scheme is `http` so we explicitly specify `https` to use the HTTPS protocol for communication between nodes.
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/howtos.md]: I thought the comma wouldn't be used before *and*. Thanks, I learned something new.

**Comment by yuce** [comment]
@jaffee Updated with your feedback.

File: docs/howtos.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+[gossip]
+seed = "01.pilosa.local:15000"
+port = 17000
+key = "pilosa.local.gossip32"
+```
+
+Here is some explanation of the configuration items:
+* `data-dir` points to the directory where the Pilosa server writes its data. If it doesn't exist, the server will create it.
+* `bind` is the address which the server listenes to for incoming requests. The address is composed of three parts, the scheme, host, and port. The default scheme is `http` so we explicitly specify `https` to use the HTTPS protocol for communication between nodes.
```
Comment (travisturner) [inline docs/howtos.md]: i would suggest (also note spelling correction in `listenes`.
```
`bind` is the address to which the server listens for incoming requests. The address is composed of three parts: scheme, host, and port. The default scheme is `http` so we explicitly specify `https` to use the HTTPS protocol for communication between nodes.
```

Comment (yuce) [reply docs/howtos.md]: @travisturner Thanks, updated.

**Pull request merged**